src/windows/views/webview.py

```diff
@@ -3156,7 +3176,7 @@ def render_cache_json(self):
         try:
             if self.window.timeline_sync and self.window.timeline_sync.timeline:
                 cache_object = self.window.timeline_sync.timeline.GetCache()
-                if not cache_object or cache_object.Count() <= 0:
```

Comment:
Now that the crash caused by empty caches is fixed in libopenshot, we actually want to render empty caches (otherwise we can't show the user when the cache is emptied)